test/functionalTest/cases/2507_csub_metadata_field/metadata_filter_existing_sub.test.DISABLED

```diff
@@ -1,259 +0,0 @@
-# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
It seems that the github heuristic to detect that a file has been renamed didn't detect the .test.DISABLED -> .test in this PR...

NTC (just a remark)